src/AutoEnrollPlugin/Service/StreamMappingService.cs

```diff
@@ -0,0 +1,198 @@
+using System;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Serilog;
+using Innovatrics.SmartFace.Integrations.AutoEnrollPlugin.Models;
+using Innovatrics.SmartFace.Integrations.AutoEnrollPlugin.Factories;
+using System.Collections.Generic;
+
+namespace Innovatrics.SmartFace.Integrations.AutoEnrollPlugin.Services
+{
+    public class StreamMappingService : IStreamMappingService
+    {
+        private readonly ILogger logger;
+        private readonly IConfiguration configuration;
+
+        public StreamMappingService(
+            ILogger logger,
+            IConfiguration configuration
+        )
+        {
+            this.logger = logger ?? throw new ArgumentNullException(nameof(logger));
+            this.configuration = configuration ?? throw new ArgumentNullException(nameof(configuration));
+        }
+
+        public ICollection<StreamMapping> CreateMappings(string streamId)
```

Comment:
Is it really `StreamMapping` it seems to me more like some sort of  `stream configuration`